xblock/fragment.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, content=None):
         self.js_init_fn = None
         self.js_init_version = None
         self.json_init_args = None
+        self.use_require_js = False
```

Comment:
I wonder if we should call this ``use_amd`` so that other runtimes can use an alternative to RequireJS if they choose.